app/scripts/templates/confirm.mustache

```diff
@@ -23,12 +23,11 @@
     <p class="verification-email-message">{{#t}}To better protect your Firefox data, we've emailed a confirmation link to %(email)s{{/t}}</p>
     {{/isSignIn}}
 
-
-    {{#isOpenGmailButtonVisible}}
+    {{#openWebmailButtonVisible}}
       <div class="button-row">
-        <a href="{{{ gmailLink }}}" class="button" target="_blank" id="open-gmail" type="button">{{#t}}Open Gmail{{/t}}</a>
+        <a href="{{{ webmailLink }}}" class="{{{webmailType}}} button" target="_blank" id="open-webmail" type="button">{{webmailButtonText}}</a>
```

Comment:
@larissagaulia instead of `class="{{{webmailType}}}` let's do : `data-webmail-type` and then consume it with $(event.target).data('webmailType') 

Ref : https://api.jquery.com/data/